homeassistant/components/compensation/__init__.py

```diff
@@ -98,6 +107,12 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
                 k: v for k, v in conf.items() if k not in [CONF_DEGREE, CONF_DATAPOINTS]
             }
             data[CONF_POLYNOMIAL] = np.poly1d(coefficients)
+            data[CONF_MINIMUM] = (
+                sorted_coefficients[0] if data[CONF_APPLY_LOWER_LIMIT] else None
+            )
+            data[CONF_MAXIMUM] = (
+                sorted_coefficients[-1] if data[CONF_APPLY_UPPER_LIMIT] else None
+            )
```

Comment:
Can't you just use `min(initial_coefficients)` and `max(initial_coefficients)` rather than sorting if that is all you are using the sorted list for?

Reply:
Yes, but same reason above, I don't want the second value to influence the result.